scripts/install_prereqs.sh

```diff
@@ -19,14 +19,26 @@
 #   - PROJECT - Firebase or Firestore
 #   - METHOD - xcodebuild or cmake; default is xcodebuild
 
+bundle install
+
 case "$PROJECT-$METHOD" in
```

Comment:
The tests below depend upon this value being `"$PROJECT-$METHOD-$PLATFORM"`, except for the `pod-lib-lint` one.

I've been thinking about standardizing on `"$PROJECT-$PLATFORM-$METHOD"` (reversing the last two). That would make the line below

`Firebase-iOS-xcodebuild`
`Firebase-*-xcodebuild`
`Firestore-*-xcodebuild`
`*-pod-lib-lint`
`Firestore-*-cmake`